### PR TITLE
Support for Additional options

### PR DESCRIPTION
**Pull request by teophilus**
I spread the props so I can get some additional option support, I can also (painfully) create each option with the proptypes etc.. if you like.

Let me know :) thanks!

### CODE REVIEW

**Comment by ShiiRochi** [comment]
I wrote my suggestions in the same named issue #2.

File: src/components/WaveForm.js [hunk 1]
```diff
@@ -1,13 +1,82 @@
...
+    hideScrollbar: false,
+    interact: true,
+    loopSelection: true,
+    maxCanvasWidth: 4000,
+    mediaControls: false,
+    mediaType: "audio",
+    minPxPerSec: 50,
+    normalize: false,
+    partialRender: false,
+    pixelRatio: window.devicePixelRatio,
```
Comment (ShiiRochi) [inline src/components/WaveForm.js]: What if someone will use it in SSR environment?

**Comment by ShiiRochi** [comment]
I think we should define default values for all props as they're not required. We should only define values for id. Providing defaults can result in a case when someone would need to override all of them to get a clear setup.

**Comment by ShiiRochi** [comment]
Also, maybe we can optimize the way we take options for WaveForm via the creation of a util function called for example `getWaveFormOptions`. Thus, main component will not be filled with a garbage text.

**Comment by teophilus** [comment]
It's throwing crazy errors for me, I'll need more than an hour to figure out why, i'm going to use your vanilla implementation til' i can get spend some time on this

**Pull request closed without merging**